### PR TITLE
html tables

### PR DESCRIPTION
**Pull request by jeesupark96**
image

### CODE REVIEW

File: html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <table>
+      <h1>Student Grade Table</h1>
```
Comment (BrettAlbright) [inline html-tables/index.html]: You want the title to be outside of the table

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update to the title!

File: html-tables/index.html (hunk 1 above)
Comment (jeesupark96) [reply html-tables/index.html]: ah okay I got confused with the caption I believe 
got it!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**